lib/utils/user.js

```diff
@@ -3,11 +3,13 @@
 import _ from 'lodash';
 import { ajax } from '../environment';
 import type { RedditAccount } from '../types/reddit';
-import { MINUTE, string } from './';
+import { MINUTE, downcast, string, regexes } from './';
 
 export const loggedInUser = _.once((): string | void => {
-	const userLink = document.querySelector('#header-bottom-right > span.user > a');
-	return userLink && !userLink.classList.contains('login-required') && userLink.textContent || undefined;
+	const link = document.querySelector('#header-bottom-right > span.user > a');
```

Comment:
If you do a `null` check here you might be able to avoid the later `link &&` code? Not entirely sure how Flow checking works though.